app.js

```diff
@@ -8,10 +9,11 @@ const app = express()
 app.use(express.json())
 
 app.get('/api/topics', getTopics)
+app.get('/api/articles/:article_id', findArticlesById)
 // app.get('/api/users', getUsers)
 // app.get('/api/articles', getArticles)
 
-app.use(invalidRouteHandler)
+app.get('/*', invalidRouteHandler)
```

Comment:
Not crucial at this stage - but an `app.all()` here would mean your `invalidRouteHandler` will kick into play regardless of the HTTP request method